_ingest-pipelines/processors/ml-inference.md

```diff
@@ -150,9 +166,173 @@ The response confirms that, in addition to the `passage_text` field, the process
 Once you have created an ingest pipeline, you need to create an index for ingestion and ingest documents into the index.
 {: .note}
 
----
+### Example ingest processor configuration for local models
+
+The following example creates an ingest pipeline for a `SENTENCE_TRANSFORMER` local model that requires an `text_docs` field, produces a `data` field for prediction in the model output path `$.inference_results.*.output.*.data`. The processor converts the `context` field into text embeddings and stores in the `context_embedding` field. The `input_map` and `output_map` mappings support standard JSON path notation for complex data structures.
+
+For local models, the `model_input` field is mandatory and you should specify the inputs and their formats you want to provide the ML Model. If there are any inputs in `model_config` map to be used by model, they should also be added to `model_input`
+
+For remote models, the `model_input` field is optional and its default value is `"{ \"parameters\": ${ml_inference.parameters} }`.
+{: .note}
+
+```json
+
+PUT /_ingest/pipeline/ml_inference_pipeline_local
+{
+  "description": "ingest reviews and generate embedding",
+  "processors": [
+    {
+      "ml_inference": {
+        "function_name": "text_embedding",
+        "full_response_path": true,
+        "model_id": "<your model id>",
```

Comment:
Better to explain which model you are using , suggest use one of pretrained models